catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/OrderRequestImpl.java

```diff
@@ -478,15 +487,15 @@ private boolean orderContainsSupportedDelivery() {
                             TarOutputStream tarOut = new TarOutputStream(tarFos)) {
                         getTar(tarOut, file);
                         try (TemporaryFileBackedOutputStream zipFos = new TemporaryFileBackedOutputStream(
-                                MAX_MEMORY_SIZE);
-                                ZipOutputStream zipOut = new ZipOutputStream(zipFos)) {
+                                MAX_MEMORY_SIZE); ZipOutputStream zipOut = new ZipOutputStream(
```

Comment:
This is interesting formatting. `ZipOutputStream` should probably be on a new line.